app.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+useHead({
+  titleTemplate: 'Nuxt HN | %s',
+  meta: [
+    { property: 'og:image', content: 'https://user-images.githubusercontent.com/904724/58238637-f189ca00-7d47-11e9-8213-ae072d7cd3aa.png' },
```

Comment:
Can we generate another image based on your latest design updates @antfu ?

Reply:
![Frame 1 (1)](https://user-images.githubusercontent.com/11247099/169022756-cdb6ef6f-1299-4ce0-8f80-81fb6e86a2e1.png)

Made a simple design of it, WDYT?

Reply:
Awesome!

Reply:
Thanks, updated!